test/unit/spec/OktaSignIn_spec.js

```diff
@@ -0,0 +1,78 @@
+/* eslint max-params:[0, 2], max-len:[2, 180] */
```

Comment:
Would let @mauriciocastillosilva-okta comment on this test. There was a specific use case around how unit tests are being used in the widget.

Reply:
so, on your case Santhosh, you were changing a model, so it is better to test the functionality of that change in a flow where it is used
But this change is for the widget object we create, which needs to return some objects/functions... I think this is good. We could add later e2e tests for each of the functions the widget object has, but I think this is a good start

Does that make sense?

Reply:
Still a little fuzzy to me :( Will sync up offline. But we're good to go on this PR!